tests/testcases/a_basics/node_port_manager_testcases.py

```diff
@@ -0,0 +1,62 @@
+from tests.testcases.base_test import BaseTest
+import unittest
+import time
+
+
+class NodePortManagerTestcases(BaseTest):
+
+    def test001_reserve_release_port_using_service_guid(self):
+
+        """
+            *Test case for reserving ports using service guid .
+            Test Scenario:
+
+            #. get a running service GUID and reserve two ports using it , should success.
+            #. release this two ports one by one , should success
+            #. make sure that the port is released by reserve another new port again
+
+        """
+        node_service = self.controller.node_manager
+        guid = node_service.service.guid
+
+        self.log('reserving two ports for one service')
+        node_port = self.controller.node_port_manager
+        ports = node_port.reserve(guid, 2).result
+        self.assertTrue(ports)
+        port_reserved_1 = ports[0]
+        port_reserved_2 = ports[1]
+
+        self.log('releasing  the 1st reserved port for one service')
+        node_port.release(guid, [port_reserved_1])
+        time.sleep(10)
+        node_port = self.controller.node_port_manager
+        for port in node_port.service.data["data"]["ports"]:
+            if port['port'] == port_reserved_1:
+                print("failed")
```

Comment:
replace print with an assertion to make test fail if port found.
can u do this check after reserving ports ? to make sure that they has been reserved